.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**Type:** bug / feature
+
+The following has been addressed in the PR:
+
+* [ ] There is a related issue
+* [ ] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
```

Comment:
I don't think the README has a code style section?

Reply:
I copied it from framework which also doesn't have one. lol

Reply:
removed